core/src/main/resources/lib/hudson/rssBar.jelly

```diff
@@ -24,22 +24,33 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:x="jelly:xml" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <div align="right" style="margin:1em">
-    <a href="${rootURL}/legend">${%Legend}</a>
-    <span style="padding-left:1em">
-      <a href="rssAll"><img src="${imagesURL}/atom.gif" border="0" alt="Feed" height="16" width="16"/></a>
+  <style type="text/css">
```

Comment:
Ok if you mean this style.css https://github.com/clouless/jenkins/blob/ui-classes-joblist-rss-links-and-legend/war/src/main/webapp/css/style.css

I can move it there